packages/nextjs/pages/api/etl/index.ts

```diff
@@ -79,17 +85,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
             projectId,
             impact_index,
           }) as IProjectScore;
-          await ProjectScore.create([projectScoreData], { session });
+          await TempProjectScore.create(projectScoreData);
```

Comment:
I think we'll also need a way to identify and save new `Projects` that we don't already have in our db. For example, if OSO sends back data for a project that we don't already have in our db, we'd create a `ProjectScore` instance for it, but not have a corresponding `Project` instance

Reply:
I will add an issue for this. It is not essential for the MVP and we are waiting to know exactly how new projects are going to be added between OSO and Agora and then us being informed.